Source/Bridge.swift

```diff
@@ -17,25 +18,30 @@ protocol Bridgable: AnyObject {
 
 /// `Bridge` is the object for configuring a web view and
 /// the channel for sending/receiving messages
+@MainActor
 public final class Bridge: Bridgable {
+    public typealias InitializationCompletionHandler = () -> Void
     weak var delegate: BridgeDelegate?
     weak var webView: WKWebView?
 
-    public static func initialize(_ webView: WKWebView) {
+    nonisolated public static func initialize(_ webView: WKWebView, completion: InitializationCompletionHandler?) {
```

Comment:
I've added an optional completion block. It came come in handy, especially in tests.

Reply:
Looks like this isn't optional, can you add a `= nil` please?

Reply:
Sure thing.